src/docs/ganache/reference/ganache-settings.md

```diff
@@ -10,13 +10,18 @@ You can change some features of the generated blockchain through the **Settings*
 
 <p class="text-center">*Accessing Ganache Settings*</p>
 
-We've divided the settings into 6 pages:
-
-* **Workspace** sets the workspace name and shows the currently linked Truffle projects. See our more detailed docs on [creating](/docs/ganache/workspaces/creating-workspaces) and [deleting](/docs/ganache/workspaces/deleting-workspaces) workspaces for more info.
-* **Server** shows details about the network connection, including hostname, port, network ID, and whether to automatically mine each transaction into a block.
-* **Accounts & Keys** sets details about the number of accounts created, and whether to use a specific mnemonic or let Ganache generate its own.
-* **Chain** sets details about the actual workings of the generated blockchain, including gas limit and gas price.
-* **Advanced** toggles Google Analytics, which is useful for the Ganache team to track usage of the application.
+We've divided the settings into several pages:
+
+* **Workspace** sets the workspace name and shows the currently linked Truffle or Corda projects. See our more detailed docs on [creating](/docs/ganache/workspaces/creating-workspaces) and [deleting](/docs/ganache/workspaces/deleting-workspaces) workspaces for more info.
+* **Ethereum**
+  * **Server** shows details about the network connection, including hostname, port, network ID, and whether to automatically mine each transaction into a block.
+  * **Accounts & Keys** sets details about the number of accounts created, and whether to use a specific mnemonic or let Ganache generate its own.
+  * **Chain** sets details about the actual workings of the generated blockchain, including gas limit and gas price.
+  * **Advanced** toggles Google Analytics, which is useful for the Ganache team to track usage of the application.
+* **Corda**
+  * **Nodes** manages the nodes for the network.
+  * **Notaries** manages the notaries for the network.
+  * **Advanced** sets the default PostgreSQL port and toggles Google Analytics, which is useful for the Ganache team to track usage of the application.
```

Comment:
which is useful for the team to improve Ganache based on anonymous metrics (maybe a link to learn more about what we collect).

Maybe put this part in parenthesis?

Reply:
I'll update the text, but don't now we don't have a process in place to notate exactly what we track, and update a site listing what we track. I'm only fine with this lack of documentation since we are open source.